ClLike/cl_like/ccl.py

```diff
@@ -63,9 +63,8 @@ def initialize_with_params(self):
     def get_requirements(self):
         # Specify A_sE9 and sigma8 in get_can_support_params to allow both
         # inputs.
-        params = {'Omega_c': None,
-                  'Omega_b': None,
-                  'h': None,
+        params = {'Omega_m': None,
```

Comment:
Ideally you would use `allow_agnostic` as in https://github.com/xC-ell/xCell-likelihoods/blob/7fcad3e14f5310d785a3b6a3c4348fe333035ebe/ClLike/cl_like/ccl_blcdm.py#L100. Then you don't need to specify the input paramters and you don't need to remove `Omega_m` from `get_can_provide_params`.